icons/vote.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+	<path d="m6 14v8h-2 18-2v-12h-3"/>
+	<path d="m2 2h11c7 0 7 4 7 4h-5c-3 0-5 2-5 2l2 2c2 2 0 4-2 2l-6-6h-2"/>
+	<path d="m10 17 2 2 4-5"/>
```

Comment:
Nice, that's a much cleaner design, I think we can go with this one, but we should definitely use the existing checkmark as it appears on `file-check`, `verified` etc.
![image](https://user-images.githubusercontent.com/17746067/216261726-dd89650d-6f7c-4869-9505-463632c8cdb8.png)

```suggestion
  <path d="m9 12 2 2 4-4" />
  <path d="M5 7c0-1.1.9-2 2-2h10a2 2 0 0 1 2 2v12H5V7Z" />
  <path d="M22 19H2" />
```

Reply:
Very good point, I will put them all together.

Reply:
I’ve already done that for you, hence the suggestion. :)

Reply:
That's great! (I thought you only added the checkmark earlier...)